products/idn/api/patch-requests.md

```diff
@@ -0,0 +1,500 @@
+---
+id: patch-requests
+title: Patch Requests
+pagination_label: Patch Requests
+sidebar_label: Patch Requests
+sidebar_position: 8
+sidebar_class_name: patchRequests
+keywords: ['patch']
+description: Read this guide to learn how to send PATCH requests to SailPoint's IdentityNow APIs. 
+tags: ['patch', 'guide']
+---
+
+## PATCH requests 
+
+You can use the IdentityNow (IDN) APIs to update existing resources. Many of the APIs offer multiple ways to do so: 
```

Comment:
We tend to avoid using "IDN" as an abbreviation in our docs. It might be common in the API docs, but I figured I'd point it out just so you have the info.

Reply:
I agree. I've been using it for a while, but I'm not really saving much space or time by abbreviating it